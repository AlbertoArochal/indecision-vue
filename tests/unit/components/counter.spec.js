import { shallowMount, mount } from "@vue/test-utils";
import Counter from "@/components/Counter.vue";

describe("Counter.vue", () => {
    it("should match snapshot", () => {
        const wrapper = shallowMount(Counter);
        const h2 = wrapper.find("h2");
        expect(h2.text()).toBe("Counter");
    });
});
