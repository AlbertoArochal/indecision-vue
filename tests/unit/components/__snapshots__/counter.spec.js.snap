// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter.vue should match snapshot 1`] = `[Function]`;
